src/api/app/controllers/webui/monitor_controller.rb

```diff
@@ -90,13 +90,14 @@ def events
     check_ajax
     data = {}
 
-    arch = params[:arch] || ''
+    arch = Architecture.find_by(name: params[:arch]) || Architecture.find_by(name: 'x86_64')
```

Comment:
[nitpick]

I would avoid duplicating `Architecture.find_by(name: ...)` and instead do:
```ruby
arch = Architecture.find_by(name: params.fetch(:arch, 'x86_64'))
```

Reply:
isn't that a different thing? The old code checks if the architecture in params exists and uses a fallback architecture - while your code only takes a fallback if the param is not there.

Just saying